src/redux/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import fetchEpisodes from 'redux/reducers/episodesReducer';
```

Comment:
Re-think name of this reducer. There are two schools: one is create a couple of reducers in file 'episodesReducer' and then import all of them here. The other one is to name this reducer inside 'episoderReducer' or whatever and keep all cases inside, then there is no need to specify what you're importing and just say:
```
import episodes from 'redux/reducers/episodesReducer';
const rootReducer = combineReducers({
  episodes
});
```

Up to you which one you'll choose. In bigger apps I prefer second (when there is more than 2-3 reducers).